### PR TITLE
Redesign Past NUANSA

### PR DESCRIPTION
**Pull request by chrisjwelly**
A redesigned Past NUANSA page based on [fullPage.js](https://github.com/alvarotrigo/fullPage.js)

### CODE REVIEW

File: app/views/past_events/index.html.erb [hunk 1]
```diff
@@ -145,7 +126,6 @@
       <div class="time">
         <h4>2008</h4>
       </div>
-    </li>
 		<div style="clear: both;"></div>
-	</ul>
-</div>
\ No newline at end of file
+</div>
+<%= javascript_include_tag('fullpage/past_nuansa.js') %>
```
Comment (chrisjwelly) [inline app/views/past_events/index.html.erb:141]: I'm not sure why, but despite adding this same line in [app/views/layouts/application.html.erb](https://github.com/nuansa-cp/gapura/pull/23/files#diff-9599427925097c3c66f26ac1e0de5cad), the javascript won't load properly. So I added this line in the .erb file as a quick fix

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -13,7 +13,6 @@
 //= require rails-ujs
 //= require activestorage
 //= require turbolinks
-//= require_tree .
 //= require jquery3
 //= require popper
-//= require bootstrap-sprockets
+//= require bootstrap-sprockets
```
Comment (stevenwjy) [inline app/assets/javascripts/application.js:18]:
```suggestion
//= require bootstrap-sprockets

```

File: app/assets/javascripts/layouts/nav-menu.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          $('body').removeClass('mobile-nav-active');
+          $('#mobile-nav-toggle i').toggleClass('fa-times fa-bars');
+          $('#mobile-body-overly').fadeOut();
+        }
+      }
+    });
+  } else if ($("#mobile-nav, #mobile-nav-toggle").length) {
+    $("#mobile-nav, #mobile-nav-toggle").hide();
+  }
+});
```
Comment (stevenwjy) [inline app/assets/javascripts/layouts/nav-menu.js]:
```suggestion
});

```

File: app/assets/stylesheets/home-components/intro.scss [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  #intro .play-btn {
+    margin-top: 30%;
+    margin-bottom: 20%;
+    visibility: hidden;
+  }
+
+  #intro .btn-container {
+    margin-bottom: 15%;
+  }
+}
```
Comment (stevenwjy) [inline app/assets/stylesheets/home-components/intro.scss]:
```suggestion
}

```

File: app/assets/stylesheets/layouts/nav-menu.css [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+  #mobile-nav-toggle {
+    display: inline;
+  }
+}
+
+@media (max-width: 1199px) {
+  .nav-menu a {
+    padding: 8px 4px;
+  }
+}
```
Comment (stevenwjy) [inline app/assets/stylesheets/layouts/nav-menu.css]:
```suggestion
}

```

**Review by stevenwjy — APPROVED** [review]

**Pull request merged**